### PR TITLE
Fix table with null values

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: core/src/main/java/org/mapfish/print/processor/jasper/TableProcessor.java [hunk 1]
```diff
@@ -291,7 +291,7 @@ public Output execute(final Input values, final ExecutionContext context) throws
     // check if there are columns with mixed int and bigdecimal values
     HashSet<String> toCorrect = new HashSet<>();
     for (Map.Entry<String, Class<?>> entry : columns.entrySet()) {
-      if (entry.getValue().equals(Integer.class)) {
+      if (entry.getValue() != null && entry.getValue().equals(Integer.class)) {
```
Comment (sebr72) [inline core/src/main/java/org/mapfish/print/processor/jasper/TableProcessor.java:294]: How about Integer.class.equals(entry.getValue()) making this null check redundant. 
Is entry != null not required ?

**Review by sebr72 — APPROVED** [review]
Is the 2nd "Report Title" at the bottom of the image not a bug?

File: core/src/main/java/org/mapfish/print/processor/jasper/TableProcessor.java (hunk 1 above)
Comment (sbrunner) [reply core/src/main/java/org/mapfish/print/processor/jasper/TableProcessor.java]: Not explicitly said in the doc of `entrySet` https://docs.oracle.com/en/java/javase/17/docs/api/jdk.httpserver/com/sun/net/httpserver/Headers.html#entrySet() but for me, it's absurd that entry is `null`...

**Comment by sbrunner** [comment]
Is the 2nd "Report Title" is part of the `pageHeader` band, and we are on a new page :-)

**Pull request merged**